spacy_streamlit/visualizer.py

```diff
@@ -192,10 +192,11 @@ def visualize_ner(
         st.warning("The parameter 'labels' should not be empty or None.")
     else:
         exp = st.expander("Select entity labels")
+        labels = list(set(labels))
```

Comment:
Thanks for the PR!

One drawback is that this doesn't preserve the order for the user-provided labels. I'm honestly not 100% sure what streamlit does here or how much it matters in the scheme of things, but how about moving this same idea to line 189 above?

Reply:
Thanks for the review. I've moved the change to line 189 as per your suggestion.